app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt

```diff
@@ -465,10 +510,22 @@ sealed class CurrentSessionErrorState {
     object SessionExpired : CurrentSessionErrorState()
 }
 
+sealed class NewClientData(open val date: String, open val deviceInfo: String) {
+    data class CurrentUser(override val date: String, override val deviceInfo: String) : NewClientData(date, deviceInfo)
+    data class OtherUser(
+        override val date: String,
+        override val deviceInfo: String,
+        val userId: UserId,
+        val userName: String?,
+        val userHandler: String?
```

Comment:
I think it's a typo and it's `userHandle` without the `r` 😄

Reply:
i really thought it's handler :D